src/qt/mAPreferencesWindow.cpp

```diff
@@ -300,21 +386,34 @@ void mAPreferencesWindow::ProbeAudioDevices()
                 ui->audioOutput->setCurrentIndex(ui->audioInput->count()-1);            
         }
     }
-    
+
 //    if( dac == 0 )
 //        ui->audioOutput->setCurrentIndex( 0 );
 //    if( adc == 0 )
 //        ui->audioInput->setCurrentIndex( 0 );
     
-    this->SelectedAudioInputChanged();
-    this->SelectedAudioOutputChanged();
+    this->selectedAudioInputChanged();
+    this->selectedAudioOutputChanged();
+}
+
+
+// 1.5.0.1 (ge) added
+void mAPreferencesWindow::selectedAudioDriverChanged()
+{
+    // get item data
+    int driver = ui->audioDriver->itemData(ui->audioDriver->currentIndex()).toInt();
+    // trigger probe
+    this->probeAudioDevices(driver);
+    // reset to first items
+    ui->audioOutput->setCurrentIndex( 0 );
+    ui->audioOutput->setCurrentIndex( 0 );
```

Comment:
```suggestion
    ui->audioInput->setCurrentIndex( 0 );
```

Reply:
can remove both of these if using `probeAudioDevices` with new `resetToDefault = true`

Reply:
done